client/gutenberg/editor/controller.js

```diff
@@ -38,7 +38,6 @@ function registerDataPlugins( userId ) {
 	const storageKey = 'WP_DATA_USER_' + userId;
 
 	use( plugins.persistence, { storageKey: storageKey } );
-	use( plugins.asyncGenerator );
```

Comment:
This has been removed from the latest version of `@wordpress/data` package: https://github.com/WordPress/gutenberg/blob/master/packages/data/CHANGELOG.md#300-2018-10-29